TestForSignUp/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium.Chrome;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    class Program
+    {
+        static void Main()
+        {
+             var browser = new ChromeDriver();
+             browser.Navigate().GoToUrl("https://localhost:44335/signup");
+             Thread.Sleep(5000);
+             var input = new InputGenerator();
+             var filler = new ElementFiller(browser);
+             filler.Fill(input.commonString, input.emailString, input.dateString);
```

Comment:
Where do we have first and last names input? 
Do we test image addition?

Reply:
1. commonString
2. not yet, but I’ll clarify

Reply:
I will add it in next pull request, if necessary